SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -3050,7 +3050,23 @@ void CIncEulerSolver::SetResidual_DualTime(CGeometry *geometry, CSolver **solver
 
       LinSysRes.AddBlock(iPoint, Residual);
 
+      // ADD old version
       if (implicit) {
+        for (iVar = 1; iVar < nVar; iVar++) {
+          if (config->GetTime_Marching() == DT_STEPPING_1ST)
+            Jacobian_i[iVar][iVar] = Volume_nP1 / TimeStep;
+          if (config->GetTime_Marching() == DT_STEPPING_2ND)
+            Jacobian_i[iVar][iVar] = (Volume_nP1*3.0)/(2.0*TimeStep);
+        }
+        for (iDim = 0; iDim < nDim; iDim++)
+          Jacobian_i[iDim+1][iDim+1] = Density*Jacobian_i[iDim+1][iDim+1];
+        Jacobian_i[nDim+1][nDim+1] = Density*Cp*Jacobian_i[nDim+1][nDim+1];
+
+        Jacobian.AddBlock2Diag(iPoint, Jacobian_i);
+      }
+      // ADD old version
+
+      if (implicit && false) {
```

Comment:
For the Jacobian change, let's go with the data: sounds like the version without the preconditioner is an order of magnitude better in your tests, so I would make it the default here and remove the old implementation from the code (remember we can always recover to from the git history). 

It is likely that adding the preconditioner portion is hurting the diagonal dominance of the system, or we have a bug / logic error that would be good to understand once there is time to do more analysis

Reply:
The version without Preconditioner is the "exact" Jacobian dR/dV imo. I recall that (artificially) multiplying with the preconditioner (Gamma*dR/dV) was s.th. we  / I / somebody tested because there is no Jacobian contribution for the continuity equation (iVar=1). 
So we could have an unlucky case where the artificial Jacobian is hurting convergence compared to the exact one?
I looked in the Paper of Smith&Weiss for additional info without much success :)

Reply:
Gonna stick my nose in :) You guys can also consider an approach similar to the associated with CENTRAL_JACOBIAN_FIX_FACTOR, i.e. multiply this Jacobian contribution with a weight on the off chance some problem might benefit from it.

Reply:
Hi Pedro, I've added an option for the fix factor in centered numerics of the incompressible methods. The preconditioner acts as the multiplication factor already, but there is now an option for an additional fix factor. For a test case I'm currently using, factor below 1 has small improvements to convergence but not a significant step change so the default value for CENTRAL_INC_JACOBIAN_FIX_FACTOR in the config is 1.0. But like you say having this option will give more tweaking flexibility in the config in case it benefits some other problems.

Reply:
That was not what I meant... maybe I can explain better this afternoon.